providers/arista/provider/provider.go

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package provider
+
+import (
+	"errors"
+	"go.mondoo.com/cnquery/providers-sdk/v1/vault"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"go.mondoo.com/cnquery/llx"
+	"go.mondoo.com/cnquery/providers-sdk/v1/inventory"
+	"go.mondoo.com/cnquery/providers-sdk/v1/plugin"
+	"go.mondoo.com/cnquery/providers-sdk/v1/upstream"
+	"go.mondoo.com/cnquery/providers/arista/connection"
+	"go.mondoo.com/cnquery/providers/arista/resources"
+)
+
+type Service struct {
+	runtimes         map[uint32]*plugin.Runtime
+	lastConnectionID uint32
+}
+
+func Init() *Service {
+	return &Service{
+		runtimes:         map[uint32]*plugin.Runtime{},
+		lastConnectionID: 0,
+	}
+}
+
+func (s *Service) ParseCLI(req *plugin.ParseCLIReq) (*plugin.ParseCLIRes, error) {
+	flags := req.Flags
+	if flags == nil {
+		flags = map[string]*llx.Primitive{}
+	}
+
+	conn := &inventory.Config{
+		Type: req.Connector,
+	}
+
+	// custom flag parsing
+	user := ""
+	port := 0
+	if len(req.Args) != 0 {
+		target := req.Args[0]
+		if !strings.Contains(target, "://") {
+			target = "ssh://" + target
+		}
+
+		x, err := url.Parse(target)
+		if err != nil {
+			return nil, errors.New("incorrect format of target, please use user@host:port")
+		}
+
+		user = x.User.Username()
+		conn.Host = x.Hostname()
+		conn.Path = x.Path
```

Comment:
Is this intended?

The error above states "please use user@host:port". Nothing about a path. It is also not used in `NewAristaConnection`.

Reply:
no, that is a copy paste bug :-)